ambari-server/src/main/java/org/apache/ambari/server/upgrade/UpgradeCatalog300.java

```diff
@@ -561,6 +561,13 @@ protected void executeDMLUpdates() throws AmbariException, SQLException {
     updateKerberosConfigurations();
     upgradeLdapConfiguration();
     createRoleAuthorizations();
+    addUserAuthenticationSequence();
+  }
+
+  protected void addUserAuthenticationSequence() throws SQLException {
+    final long maxUserAuthenticationId = fetchMaxId(USER_AUTHENTICATION_TABLE, "user_authentication_id");
```

Comment:
Please use `USER_AUTHENTICATION_USER_AUTHENTICATION_ID_COLUMN` instead of `"user_authentication_id"`.

Reply:
Done